content/review.rst

```diff
@@ -192,16 +192,32 @@ Merge your branch
 After the merge request and review was approved you need to merge your branch
 into master.
 
-After your review request was approved, you can send your branch to PQM
-for automatic testing and merging.
-Use the following command::
+After your review request was approved, you can send your branch to
+and conduct a test_review::
 
-    paver pqm GITHUB_PULL_REQUEST_ID
+    paver test_review
```

Comment:
we now need... for the coverage report :)

`paver test_review GITHUB_PULL_REQUEST_ID`